### DISCUSSION TITLE
(Bug report) Unable to Use api.triggerCommand

### DISCUSSION THREAD

**Discussion by BeatLink** [TriliumNext/Trilium#7808] [Q&A] [answered]
### Trilium Version

0.60.4

### What operating system are you using?

Other Linux

### What is your setup?

Local + server sync

### Operating System Version

Linuxt Mint Cinnamon

### Description

I'm trying to create a sidebar launcher script to move notes up and down from mobile. Executing the below JS frontend script does nothing however. No errors show either. Is something wrong with my code? 

```js
async function moveNoteUp() {    
    let currentNote = await api.getActiveContextNote();
    api.log(currentNote)
    await api.triggerCommand("moveNoteUp", currentNote)
}

moveNoteUp();
```

**Comment by zadam** [comment, marked as answer]
`api.triggerCommand()` is a low-level API and is not really recommended to use. 

"moveNoteUp" expected a FancytreeNode object and is not handled in the global scope (which in effect means it can't be triggered from scripts).

Unfortunately I don't think there's a way to achieve what you're trying to do using API, without shuffling the entities mauanlly in the backend.

**Comment by BeatLink** [comment]
Thanks, looks like I will need to manually implement it
